src/Model/AuditReader.php

```diff
@@ -47,8 +47,21 @@ public function find(string $className, $id, $revisionId): ?object
         }
     }
 
+    /**
+     * NEXT_MAJOR: Change the default limit value to `null`.
```

Comment:
Consistent with https://github.com/sonata-project/SonataAdminBundle/pull/7959

Reply:
Why allow null? should the change be only the deprecation of null?

Reply:
What if you want to get all the revision history ?

The auditReader allow to use null limit, so why not keeping the feature ?

Reply:
OH I see, the deprecation is for offset, and the allow null is for limit. 👍